INSTALL.md

```diff
@@ -7,116 +7,135 @@
 Installing OpenCoarrays
 =======================
 
-[![Download as PDF][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)
+[The following link is commented until it can be updated. To uncomment delete the backslashes, the ourtermost square brackets, and the trailing colon and hash mark]:#
+[\[\!\[Download as PDF]\][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)]:#
```

Comment:
Also, this fails to correctly comment out the link: https://github.com/sourceryinstitute/opencoarrays/blob/51a047f53110b0c28f944ec0b2c917c283b9b84c/INSTALL.md